policyengine_us/variables/gov/states/oh/tax/income/credits/oh_senior_citizen_credit.py

```diff
@@ -20,11 +20,12 @@ def formula(tax_unit, period, parameters):
         has_not_taken_lump_sum_distribution = person(
             "oh_has_not_taken_oh_lump_sum_credits", period
         )
-        age = person("age", period)
 
-        any_elderly = tax_unit.any(age >= p.age_threshold)
+        age = person("age", period)
+        any_elderly = age >= p.age_threshold
```

Comment:
any implies tax unit
```suggestion
        elderly = age >= p.age_threshold
```